docs/source/repak/assets/texture.rst

```diff
@@ -0,0 +1,85 @@
+Textures
+^^^^^^^^
+
+Textures are the foundation of some RPak asset types.
+They cannot be used directly by the game, but are instead referenced by other asset types which the game can use by itself.
+
+The image used by a texture must be in the .dds format and must be in one of the following compression types:
+
+- BC1 SRGB
+- BC2 SRGB
+- BC3 SRGB
+- BC7
+- BC7 SRGB
+- DXT1
+- DXT3
+- DXT5
+- BC4U
+- BC5U UNORM
+
+.. warning::
+    SRGB DDS compression types are preferred, as they can prevent the texture's colour from looking "washed out"
+
+Examples:
+=========
+
+1. Basic Texture Asset
+----------------------
+
+.. code-block:: json
+
+    {
+        "$type": "txtr",
+        "path": "textures/models/humans/test_texture",
+        "disableStreaming": true
+    }
+
+.. note::
+    The image file in this texture asset will be called ``test_texture.dds`` and will be at ``<ASSETSDIR>/textures/models/humans/test_texture.dds``
+
+.. note::
+    This texture will not be stored in a .starpak file, and all mip levels will be stored in the .rpak file
+
```

Comment:
What does this mean, and why is it relevant? Is this important enough to note while potentially confusing the reader?

Reply:
I'd say that it's kinda important that the texture won't use a starpak file, I could take or leave the bit specifying mip levels

Reply:
As for what this means, it means that (if no other textures use the starpak file) no starpak file will be created at all, which is kinda good atm because starpak loading is currently all relative to the r2/paks folder